.gitignore

```diff
@@ -7,3 +7,4 @@
 /captures
```

Comment:
Hi, 
please add some more ignores to this gitignore file. 
You are uploading 12 000 files. I can't review this.
Please consider adding:

/**/.DS_Store
/**/*.iml
/**/.idea
/**/.project
/**/.settings
/**/gen
/**/gen-external-apklibs
/**/out
/**/.classpath
/**/bin
/**/target
/**/tmp
/**/atlassian-ide-plugin.xml

.gradle
/local.properties
/**/local-modules.properties
/.idea/workspace.xml
/.idea/libraries
.DS_Store
/build
/captures
.externalNativeBuild